stable/storageos/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: storageos
 version: 1.1.4
```

Comment:
This needs to be changed as the CI/CD logs, at the moment we are going to wait for the new image to be released first before moving forward